installing/acs-installation-platforms.adoc

```diff
@@ -1,14 +1,144 @@
 :_content-type: ASSEMBLY
 [id="acs-installation-platforms"]
-= Installation platforms and methods
+= Supported platforms and installation methods
 include::modules/common-attributes.adoc[]
 :context: acs-installation-platforms
-
 toc::[]
 
 [role="_abstract"]
-{product-title} and {product-title-managed} are supported on various platforms, such as Red Hat OpenShift and Kubernetes platforms. 
+{rh-rhacs-first} and {product-title-managed} ({product-title-managed-short}) are supported on various platforms, such as {ocp} and Kubernetes platforms. 
```

Comment:
**issue:** ... are supported on various platforms, such as {ocp} and Kubernetes. 

Remove `platforms`.

Reply:
Done